backend/src/main/java/de/neuefische/app/spotify/SpotifyApiController.java

```diff
@@ -0,0 +1,60 @@
+package de.neuefische.app.spotify;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.*;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Base64;
+
+@RestController
+@RequestMapping("/callback")
```

Comment:
```suggestion
@RequestMapping("/api/callback")
```

consider adding api or another prefix to your backend requests. With this prefix you'll have no conflicts with your Frontend-Router and can use the same url for the Frontend, without the prefix (frontend /users for userinfos, backend /api/users for userinfos)